README.md

```diff
@@ -113,6 +114,44 @@ Get pandas dataframe by taking the name
 r.get_pandasdf("pandasdf")
 ```
 
+### Files <a name="files"></a>
+```python
+#Example
+Add file from path
+test_file = os.path.join(os.path.dirname(__file__), "ideal-engineer.png")
```

Comment:
I think we should make our documentation a little bit more generalized. We can probably remove the `os.join.path` as well from the documentation.
```python
# remove
Add file from path
test_file = os.path.join(os.path.dirname(__file__), "ideal-engineer.png")

# Replace
r.add_files_from_path("TestFile", test_file)
# With something along the lines of:
r.add_files_from_path("filename.txt", test_file)
```
Something like this should be fine. The goal is to be explicitly clear about how to add a file without adding code that distracts from what it is we are trying to convey.